### PR TITLE
feat: add search bar to index page

### PR DESCRIPTION
**Pull request by leangseu-edx**
Adding search bar to index page.
## Screenshot-1: Normal State

Screen Shot 2021-08-11 at 9 13 04 PM

## Screenshot-2: Active State
Screen Shot 2021-08-11 at 9 13 24 PM

## Screenshot-2: Search Results
Screen Shot 2021-08-11 at 9 13 58 PM

### CODE REVIEW

**Review by nsprenkle — COMMENTED** [review]
Can you provide info on how to verify this locally/in a sandbox? Or, even better, a sandbox/stage location where these changes are already live?

**Comment by awaisdar001** [comment]
> Can you provide info on how to verify this locally/in a sandbox? Or, even better, a sandbox/stage location where these changes are already live?

Sure, just clone the repo, checkout the branch and open the `index.html` by double clicking on it. 

Screen Shot 2021-08-12 at 9 57 16 PM

File: index.html [hunk 1]
```diff
@@ -56,6 +57,12 @@ <h2 class="header__tagline">Documentation for edx.org and the Open edX Community
       <div class="content content--view">
 
         <article class="content__main">
+          <div class="search-description__copy">
```
Comment (nsprenkle) [inline index.html:60]: Now that this is here, you should probably remove the `Search edX Documentation` section from later down the page.

Comment (awaisdar001) [reply index.html:60]: Sure, makes sense.

**Comment by awaisdar001** [comment]
@nsprenkle Can you take an other look?

**Review by mehaknasir — APPROVED** [review]

**Review by awaisdar001 — APPROVED** [review]

**Pull request merged**